xdsl/dialects/mpi.py

```diff
@@ -328,6 +346,22 @@ def get(cls):
         return cls.build(result_types=[i32])
 
 
+@irdl_op_definition
+class CommSize(MPIBaseOp):
+    """
+    Represents the MPI_Comm_size(MPI_Comm comm, int *size) function call which returns the size of the communicator
```

Comment:
Looks like this needs an actual argument. Is this going to be added later?

Reply:
We don't model anything except `MPI_COMM_WORLD` currently as it would be a giant PITA. I thought I added that everywhere, I guess I didn't.

Reply:
Ah, so there is only this communicator and hence you don't need an operand. Makes sense :+1: